packages/twilio-run/src/serverless-api/utils.ts

```diff
@@ -49,20 +55,22 @@ export async function getFunctionServiceSid(
   }
 
   debug('Could not determine existing serviceSid');
+  debug(`${username}:${region}`);
   return undefined;
 }
 
 export async function saveLatestDeploymentData(
```

Comment:
It looks like you are missing in the `packages/twilio-run/src/commands/deploy.ts` file an update to the call of `saveLatestDeploymentData` that passes `config.region`.